core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -80,6 +80,22 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator with UserIn
         }
       }
     } ~
+    path("workspaces" / Segment / Segment / "catalog" ) { (workspaceNamespace, workspaceName) =>
+      get {
+        requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
+                                WorkspaceService.GetCatalog(WorkspaceName(workspaceNamespace, workspaceName)))
+      }
+    } ~
+    path("workspaces" / Segment / Segment / "catalog" ) { (workspaceNamespace, workspaceName) =>
+      patch {
+        requireUserInfo() { userInfo =>
```

Comment:
This directive is not needed because `userInfo` is already available to all routes, thanks to line 24.

Actually would you mind removing similar lines throughout this file?